### ISSUE TITLE
ConvTranspose2d onnxruntime and pytorch forward results are inconsistent

### ISSUE THREAD

**Issue by mi-tao** [microsoft/onnxruntime#14208]
### Describe the issue

Setting the parameters outpadding<=kernel_size and outpadding<dilation in pytorch causes inconsistencies

### To reproduce

weight: All initialized to 1
The input is: 0.5
PyTorch results:
![图片](https://user-images.githubusercontent.com/81147852/211509585-e45b6fd9-440b-4c0c-b707-55dfe0cbc466.png)
ort results:
![图片](https://user-images.githubusercontent.com/81147852/211511384-7832ad12-7b13-4cc3-ae42-9b84db2c3709.png)

### Urgency

Very, very important

### Platform

Linux

### OS Version

ubuntu 18.04

### ONNX Runtime Installation

Built from Source

### ONNX Runtime Version or Commit ID

onnxruntime-linux-x64-1.13.1

### ONNX Runtime API

C++

### Architecture

X64

### Execution Provider

Default CPU

### Execution Provider Library Version

ptorch 1.10.0+cu113

**Comment by xadupre** [comment]
Is it possible to know the exact instructions you executed with torch? If possible, the onnx model you got after conversion would help as well.

**Comment by qiangxinixn** [comment]
> 

` torch.nn.ConvTranspose2d(1,out_channels=3,kernel_size=2,stride=1,padding=3,dilation=2,output_padding=1)`

[conv_transpose.zip](https://github.com/microsoft/onnxruntime/files/10388109/conv_transpose.zip)
thank u

**Comment by qiangxinixn** [comment]
> Is it possible to know the exact instructions you executed with torch? If possible, the onnx model you got after conversion would help as well.

` torch.nn.ConvTranspose2d(1,out_channels=3,kernel_size=2,stride=1,padding=3,dilation=2,output_padding=1)`

[conv_transpose.zip](https://github.com/microsoft/onnxruntime/files/10388109/conv_transpose.zip)
thank u
